base/pkg/github.jl

```diff
@@ -10,7 +10,8 @@ const AUTH_DATA = Dict{Any,Any}(
 )
 
 function user()
-    if !success(`git config --global github.user`)
+    usr = LibGit2.lookup(String, LibGit2.GitConfig(), "github.user")
```

Comment:
In the LibGit2.jl library you should be able to use `LibGit2.GitConfig["github.user"]`, it may be too much code to transfer the whole config business over.